LibreNMS/OS/Junos.php

```diff
@@ -82,6 +87,58 @@ public function pollOS(DataStorageInterface $datastore): void
         }
     }
 
+    public function discoverWirelessRsrp()
+    {
+        $sensors = [];
+        $oids = SnmpQuery::walk('JUNIPER-WIRELESS-WAN-MIB::jnxWirelessWANNetworkInfoRSRP')->table(1);
+        $oid = '.1.3.6.1.4.1.2636.3.87.1.1.1.1.32'; // jnxWirelessWANNetworkInfoRSRP
+
+        foreach ($oids as $index => $entry) {
+            $sensors[] = new WirelessSensor('rsrp', $this->getDeviceId(), $oid . $index, 'junos', $index, 'Modem RSRP', null, 1, 1, 'sum', null, null, -100);
+        }
+
+        return $sensors;
+    }
+
+    public function discoverWirelessRsrq()
+    {
+        $sensors = [];
+        $oids = SnmpQuery::walk('JUNIPER-WIRELESS-WAN-MIB::jnxWirelessWANNetworkInfoRSRQ')->table(1);
+        $oid = '.1.3.6.1.4.1.2636.3.87.1.1.1.1.33'; // jnxWirelessWANNetworkInfoRSRQ
+
+        foreach ($oids as $index => $entry) {
+            $sensors[] = new WirelessSensor('rsrq', $this->getDeviceId(), $oid . $index, 'junos', $index, 'Modem RSRQ', null, 1, 1, 'sum', null, null, -20);
```

Comment:
You should add the current value and you can skip arguments with defaults by setting arguments by name like `low_limit: -20`

Reply:
FYI, the current value is here: `$entry['JUNIPER-WIRELESS-WAN-MIB::jnxWirelessWANNetworkInfoRSRQ']`

Reply:
Thanks, updated the code. Will still need to access a lab device for the test data.